homeassistant/components/media_player/const.py

```diff
@@ -63,3 +64,40 @@
 SUPPORT_PLAY = 16384
 SUPPORT_SHUFFLE_SET = 32768
 SUPPORT_SELECT_SOUND_MODE = 65536
+
+MEDIA_KEYS = [
```

Comment:
these must be reduced to the common denominator (probably numbers + prog + volume + power)